apps/docs/src/content/docs/Example.svelte

```diff
@@ -25,60 +28,100 @@
 		elements = elements;
 	}
 
-	$: example_task = task(fn, { max: 1, kind: selectedTaskType });
+	const stop_after = 3000;
+	const now = readable(Date.now(), (set) => {
+		const interval = setInterval(() => {
+			set(Date.now());
+		}, 100);
+
+		return () => clearInterval(interval);
+	});
```

Comment:
why use a store here instead of a normal variable?

Reply:
I couldn't get a normal variable working for this. It wouldn't update consistently